splinterd/api/static/openapi.yml

```diff
@@ -599,6 +599,457 @@ paths:
         500:
           description: Internal service error
 
+  /biome/register:
+    post:
+      tags:
+        - Biome
+      description: Create new Splinter User with username and password credentials
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                username:
+                  description: username of a new user
+                hashed_password:
+                  description: |
+                     Hashed password to be used for user authentication.
+              required:
+                - username
+                - hashed_password
+              example:
+                username: Anne@AcmeCorp.com
```

Comment:
capital letters should be avoided in email addresses. `alice@acme.com` would be a good choice.

Reply:
repeated below